cmd/oras/pull.go

```diff
@@ -148,9 +148,29 @@ func runPull(opts pullOptions) error {
 		return ret, nil
 	}
 
+	ctx, _ := opts.SetLoggerLevel()
+	var dst = file.New(opts.Output)
+	dst.AllowPathTraversalOnWrite = opts.PathTraversal
+	dst.DisableOverwrite = opts.KeepOldFiles
+
 	pulledEmpty := true
 	copyOptions.PreCopy = display.StatusPrinter("Downloading", opts.Verbose)
 	copyOptions.PostCopy = func(ctx context.Context, desc ocispec.Descriptor) error {
+		// restore successors, if applicable
+		ss, err := content.Successors(ctx, dst, desc)
+		if err != nil {
+			return err
+		}
+		for _, s := range ss {
+			exists, _ := dst.Exists(ctx, s)
+			if exists {
+				if _, loaded := printed.LoadOrStore(display.Name(s), true); !loaded {
+					if err = display.PrintStatus(s, "Restored   ", opts.Verbose); err != nil {
+						return err
+					}
+				}
+			}
```

Comment:
The missing else block here means: a manifest is pulled but one of its successor is not, which is a big SDK error and should be handled.

Reply:
This `exists` check is removed per the next comment, so this comment is resolved.